src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
-}
+  const LONG_TERM = 7;
+  const LONG_TERM_DISCOUNT = 50;
```

Comment:
i would suggest to add one more const with discount for short-term rental and one const with basic price daily.

Reply:
Correct! Add another const